<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::DockOption::AllowTabbedDocks|QMainWindow::DockOption::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="timeIntervallBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Time Intervall</string>
    </property>
    <widget class="QDateTimeEdit" name="startTime">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="endTime">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="startLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>63</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Startime:</string>
     </property>
    </widget>
    <widget class="QLabel" name="endLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>63</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Endtime:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="generationTypeBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Generation Process Type</string>
    </property>
    <widget class="QListWidget" name="listGenerationType">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="countryBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Country</string>
    </property>
    <widget class="QFontComboBox" name="chooseCountryBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>221</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="valueBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Value</string>
    </property>
    <widget class="QListWidget" name="listRequestValues">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="applySettingsBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Settings</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="viewBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>881</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QWidget" name="plotWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>841</width>
       <height>581</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
